core-rust/core-api-server/src/core_api/conversions/receipt.rs

```diff
@@ -170,55 +170,59 @@ pub fn to_api_next_epoch(
 
 #[tracing::instrument(skip_all)]
 pub fn to_api_fee_summary(
-    context: &MappingContext,
+    _context: &MappingContext,
     fee_summary: FeeSummary,
 ) -> Result<models::FeeSummary, MappingError> {
     Ok(models::FeeSummary {
         cost_unit_price: to_api_decimal(&fee_summary.cost_unit_price),
         tip_percentage: to_api_u16_as_i32(fee_summary.tip_percentage),
         cost_unit_limit: to_api_u32_as_i64(fee_summary.cost_unit_limit),
-        cost_units_consumed: to_api_u32_as_i64(fee_summary.total_cost_units_consumed),
+        cost_units_consumed: to_api_u32_as_i64(fee_summary.execution_cost_sum),
         xrd_total_execution_cost: to_api_decimal(&fee_summary.total_execution_cost_xrd),
         xrd_total_royalty_cost: to_api_decimal(&fee_summary.total_royalty_cost_xrd),
         xrd_total_tipped: to_api_decimal(&Decimal::ZERO),
-        xrd_vault_payments: fee_summary
-            .vault_payments_xrd
-            .map(|vault_payments| {
-                vault_payments
-                    .into_iter()
-                    .map(|(vault_id, amount)| {
-                        Ok(models::VaultPayment {
-                            vault_entity: Box::new(to_api_entity_reference(RENodeId::Object(
-                                vault_id,
-                            ))?),
-                            xrd_amount: to_api_decimal(&amount),
-                        })
-                    })
-                    .collect::<Result<_, _>>()
-            })
-            .transpose()?,
         cost_unit_execution_breakdown: fee_summary
-            .execution_cost_unit_breakdown
+            .execution_cost_breakdown
             .into_iter()
             .map(|(key, cost_unit_amount)| (key.to_string(), to_api_u32_as_i64(cost_unit_amount)))
             .collect(),
-        cost_unit_royalty_breakdown: fee_summary
-            .royalty_cost_unit_breakdown
-            .into_iter()
-            .filter_map(|(receiver, cost_unit_amount)| {
-                let global_address = match receiver {
-                    RoyaltyReceiver::Package(address) => Address::Package(address),
-                    RoyaltyReceiver::Component(RENodeId::GlobalObject(address)) => address,
-                    _ => return None,
-                };
-                let payment = models::RoyaltyPayment {
-                    royalty_receiver: Box::new(
-                        to_global_entity_reference(context, &global_address).ok()?,
-                    ),
-                    cost_unit_amount: to_api_u32_as_i64(cost_unit_amount),
-                };
-                Some(payment)
-            })
-            .collect(),
+        // TODO: Add these back when the (I think accidental?) removal in this PR is reverted:
```

Comment:
so I guess this is happening much later?
(just wanted to confirm this isn't omission)

Reply:
Thanks! Nice spot. I've picked this up with Yulong and fixed this now :)